docs/static-file-caching.md

```diff
@@ -0,0 +1,13 @@
+# Static File Caching
+
+All static files on Altis Cloud are cached on the CDN for a long period of time. This ensures the best performance when downloading static assets such as images, JavaScript and CSS files. Due to the aggressive caching times, when files are changed it is necessary reference the files by a new URL that will cause a different version to be saved on the CDN.
```

Comment:
Can/should we give the precise times here?

Reply:
I think it's ok to just be long-term. If we want to change the specific time that gives us more freedom. From a n=dev perspective the important thing is "it's a long time"